src/fsharp/FSharp.Compiler.Private.Scripting/FSharpScript.fs

```diff
@@ -83,3 +83,4 @@ type FSharpScript(?captureInput: bool, ?captureOutput: bool, ?additionalArgs: st
             stdin.Dispose()
             stdout.Dispose()
             stderr.Dispose()
+            fsi.Dispose()
```

Comment:
This will have to be casted to `IDisposable` in order to call dispose.
`(fsi :> IDisposable).Dispose()`